kafka-avro-adapter/src/kafka_consumer.cpp

```diff
@@ -142,6 +142,13 @@ KafkaConsumer::KafkaConsumer(const Options& options):
     std::string errstr;
     conf->set("metadata.broker.list", m_options.broker, errstr);
     conf->set("group.id", m_options.group, errstr);
+
+    if (!m_options.debug.empty()) {
+        if (conf->set("debug", m_options.debug, errstr) != RdKafka::Conf::CONF_OK) {
+        throw AdapterError("Failed to set an librdkafka option: " + errstr);
```

Comment:
Indentation seem to be off.
```suggestion
            throw AdapterError("Failed to set an librdkafka option: " + errstr);
```